graph-builder/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["Alex Crawford <crawford@redhat.com>"]
 
 [dependencies]
-actix-web = "^0.7.8"
+actix-web = "^0.7.12"
```

Comment:
Why does this need to be bumped?

Reply:
It doesn't, I was experimenting with transitive dependencies at one point. Should I roll

Reply:
Yes. Or update all of the dependencies in another commit.